package.json

```diff
@@ -24,6 +24,7 @@
     "polished": "^3.4.2"
   },
   "devDependencies": {
+    "@svgr/webpack": "^5.5.0",
```

Comment:
this package has a crazy amount of dependencies but it should be ok because it's installed only for generating docs